language-support/hs/bindings/test/DA/Ledger/Tests.hs

```diff
@@ -543,6 +544,13 @@ tValueConversion withSandbox = testCase "tValueConversion" $ run withSandbox $ \
     [RecordField{label="owner"},RecordField{label="bucket",fieldValue=bucketReturned}] <- return fields
     liftIO $ assertEqual "bucket" bucket (detag bucketReturned)
 
+tMeteringReport :: SandboxTest
```

Comment:
For other commands we test both at the Haskell bindings level as well as in the command. It might make sense to do that here as well.

Reply:
I wasn't sure if the test in `DA.Daml.Helper.Test.Ledger` use the Grpc or the Http protocol (or both).  I've added into the ticket that adds Http support so also add a command based test, see https://digitalasset.atlassian.net/browse/DPP-855

Reply:
sounds good